### ISSUE TITLE
some confusions

### ISSUE THREAD

**Issue by Holmes-GU** [siddhanthaldar/BAKU#13]
Hi,

Thanks for sharing the codes. 

When I use 'python train.py agent=baku suite=libero dataloader=libero suite/task=libero_90 suite.hidden_dim=256' to train the model, it outputs 'the number of parameters: 6.53M' which is smaller than 10M parameters reported in the paper. Besides, the loss trends to be negative. Do I misunderstand anything?
<img width="303" alt="image" src="https://github.com/user-attachments/assets/63621c65-610d-4b8f-8224-75e6cdb7a12e">
<img width="540" alt="image" src="https://github.com/user-attachments/assets/04fd6b08-626c-4967-9491-8fbcd77e328e">

**Comment by siddhanthaldar** [comment]
Hi,

This is the size of the transformer backbone (printed inside the [GPT class](https://github.com/siddhanthaldar/BAKU/blob/80bc41b43300f547bcbb3b7dd91e276b1dfa83b0/baku/agent/networks/gpt.py#L185)). Thank you for pointing this out. 

Regarding losses being negative, it is because we use the [negative log probability](https://github.com/siddhanthaldar/BAKU/blob/80bc41b43300f547bcbb3b7dd91e276b1dfa83b0/baku/agent/networks/policy_head.py#L48-L49) as the loss.

Hope that helps! Let me know if you have any other questions.

**Comment by siddhanthaldar** [comment]
Closing issue due to inactivity. Feel free to open again for further questions.
